content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/cloud-account-associations/index.md

```diff
@@ -260,10 +261,144 @@ chainctl iam groups remove-aws
+
+```
+gcloud iam roles list --project <your project name>
+```
+```
+---
+description: Chainguard signer role to list certificates from a CA
+etag: BwX4lw5KpDY=
+name: projects/<your project name>/roles/chainguardSignerCA
+title: Custom Role Chainguard Signer CA
+```
+
+### Google Workload Identity Pool
+
+Since Enforce uses OIDC for authentication, it relies on Workload Identity Federation to generate short-lived tokens. You can examine the identity pool, and the provider that Enforce creates with the following commands:
+
+```
+ gcloud iam workload-identity-pools list --location=global --project <your project name>
+ ```
+ ```
+---
+description: Identity pool for Chainguard impersonation.
+displayName: Chainguard Pool
+name: projects/<your project number>/locations/global/workloadIdentityPools/chainguard-pool
+state: ACTIVE
+```
+
+### Google Workload Identity Pool Provider
+
+To inspect the Workload Identity Pool Provider that Enforce creates, use the following command:
+```
+gcloud iam workload-identity-pools providers list --workload-identity-pool=projects/<your project number>/locations/global/workloadIdentityPools/chainguard-pool --location=global --project <your project name>
+```
+```
+---
+attributeMapping:
+  attribute.sub: assertion.sub
+  google.subject: assertion.sub
+description: This is the provider for impersonation by the https://issuer.enforce.dev
+  Chainguard environment's issuer for https://issuer.enforce.dev.
+displayName: chainguard-provider
+name: projects/<your project number>/locations/global/workloadIdentityPools/chainguard-pool/providers/chainguard-provider
+oidc:
+  allowedAudiences:
+  - google
+  issuerUri: https://issuer.enforce.dev
+state: ACTIVE
+```
+
+### Google Service Accounts
+
+As noted in the beginning of this guide, Enforce creates a number of service accounts in your Google project. You can list the accounts with the following command:
+
+```
+gcloud iam service-accounts list --project <your project name>
+```
+```
+DISPLAY NAME  EMAIL                                                                     DISABLED
+. . .
+              chainguard-agentless@<your project name>.iam.gserviceaccount.com       False
+              chainguard-canary@<your project name>.iam.gserviceaccount.com          False
+              chainguard-ingester@<your project name>.iam.gserviceaccount.com        False
+              chainguard-cosigned@<your project name>.iam.gserviceaccount.com        False
+              chainguard-discovery@<your project name>.iam.gserviceaccount.com       False
+              chainguard-enforce-signer@<your project name>.iam.gserviceaccount.com  False
+. . .
+```
+
+
+
+### AWS OIDC Provider
+
+Agentless Enforce with AWS creates an OIDC identity provider.
+
+You can examine the created identity provider with the following `aws` command:
+
+```
+aws iam get-open-id-connect-provider --open-id-connect-provider-arn arn:aws:iam::<your account id>:oidc-provider/issuer.enforce.dev

```

Comment:
I would add a line break or two here so that it doesn't require horizontal scrolling